File: DayTwo/Java/ceaserCipher.java
public class ceaserCipher{

    public static String ceaserCipherEncryption(String message, int shiftKey){
    
        String encryptedMessage = "";
        String capitalAlphabets = "";
        String lowerAlphabets = "";
        
        for (int number = 65; number <= 90; number++){

            capitalAlphabets += (char) number;  
            
        }   
        
        for (int integer = 97; integer <= 122; integer++){

            lowerAlphabets += (char) integer;  
            
        }
    
        for (int index = 0; index < message.length(); index++){
        
            for (int indexes = 0; indexes < capitalAlphabets.length(); indexes++){
        
                if (Character.isLetter(message.charAt(index))){
                
                    if (message.charAt(index) == capitalAlphabets.charAt(indexes)){
                    
                        encryptedMessage += capitalAlphabets.charAt(((indexes + shiftKey) % 26));
                    
                    }
                }
                
             }  
             
             for (int indexes = 0; indexes < lowerAlphabets.length(); indexes++){
        
                if (Character.isLetter(message.charAt(index))){
                
                    if (message.charAt(index) == lowerAlphabets.charAt(indexes)){
                    
                        encryptedMessage += lowerAlphabets.charAt(((indexes + shiftKey) % 26));
                    
                    }
                
                }   
        
             }
            
            if (!Character.isLetter(message.charAt(index)))
                encryptedMessage += message.charAt(index);
            
        
        }return encryptedMessage;

    
    }
    


    public static String ceaserCipherDecryption(String encryptedMessage, int shiftKey){
    
        String decryptedMessage = "";
        String capitalAlphabets = "";
        String lowerAlphabets = "";
        
        for (int number = 65; number <= 90; number++){

            capitalAlphabets += (char) number;  
            
        }   
        
        for (int integer = 97; integer <= 122; integer++){

            lowerAlphabets += (char) integer;  
            
        }
    
        for (int index = 0; index < encryptedMessage.length(); index++){
        
            for (int indexes = 0; indexes < capitalAlphabets.length(); indexes++){
        
                if (Character.isLetter(encryptedMessage.charAt(index))){
                
                    if (encryptedMessage.charAt(index) == capitalAlphabets.charAt(indexes)){
                    
                        decryptedMessage += capitalAlphabets.charAt(((indexes - shiftKey) % 26));
                    
                    }
                
                }    
        
            }
            
            for (int indexes = 0; indexes < lowerAlphabets.length(); indexes++){
        
                if (Character.isLetter(encryptedMessage.charAt(index))){
                
                    if (encryptedMessage.charAt(index) == lowerAlphabets.charAt(indexes)){
                    
                        decryptedMessage += lowerAlphabets.charAt(((indexes - shiftKey) % 26));
                    
                    }
                
                }    
        
            }
            
            if (!Character.isLetter(encryptedMessage.charAt(index)))
                decryptedMessage += encryptedMessage.charAt(index);
            
        
        }return decryptedMessage;

    
    }



}

File: DayTwo/Python/ceaser_cipher.py
def ceaser_cipher_encryption(message, shift_key):

    encrypted_message = ""
    capital_alphabets = ""
    lower_alphabets = ""
    
    for number in range(65,91):

        capital_alphabets += chr(number)  
        
    for integer in range(97,123):

        lower_alphabets += chr(integer) 
        

    for letter in message:

        for capital_character in capital_alphabets:
        
            if letter.isalpha() and letter == capital_character:
            
                encrypted_message += capital_alphabets[((capital_alphabets.index(capital_character) + shift_key) % 26)]
           
        for lower_character in lower_alphabets:
        
            if letter.isalpha() and letter == lower_character:
            
                encrypted_message += lower_alphabets[((lower_alphabets.index(lower_character) + shift_key) % 26)]  
        if not letter.isalpha():
                encrypted_message += letter
                
    return encrypted_message
    


def ceaser_cipher_decryption(encrypted_message, shift_key):

    decrypted_message = ""
    capital_alphabets = ""
    lower_alphabets = ""
    
    for number in range(65,91):

        capital_alphabets += chr(number)
        
    for integer in range(97,123):

        lower_alphabets += chr(integer)  

    for letter in encrypted_message:

        for capital_character in capital_alphabets:
        
            if letter.isalpha() and letter == capital_character:
            
                decrypted_message += capital_alphabets[((capital_alphabets.index(capital_character) - shift_key) % 26)]
                
        for lower_character in lower_alphabets:
        
            if letter.isalpha() and letter == lower_character:
            
                decrypted_message += lower_alphabets[((lower_alphabets.index(lower_character) - shift_key) % 26)] 
             
        if not letter.isalpha():
                decrypted_message += letter
                
    return decrypted_message
    
